examples/csharp-change-collection-type/README.md

```diff
@@ -0,0 +1,17 @@
+# TODO: Your example title
+
+TODO: Your example description
```

Comment:
The descriptions need to be updated as well

Reply:
Done